pkg/scheduler/cache/cache.go

```diff
@@ -325,8 +340,9 @@ func (cache *schedulerCache) processPodResourcesResizeRequest(newPod *v1.Pod) er
 					glog.Errorf("Pod %s resources restore map json marshal failed. Error: %v", newPod.Name, err)
 					return err
 				} else {
-					newPod.ObjectMeta.Annotations[api.AnnotationResizeResourcesPrevious] = string(jsonStr)
+					newPod.ObjectMeta.Annotations[api.AnnotationResizeResourcesActionVer] = string(newPod.ObjectMeta.ResourceVersion)
```

Comment:
who is updating this? newPod.ObjectMeta.ResourceVersion

Reply:
this is likely incremented by apiserver. it is always increasing across all resource objects. most likely a global counter.